src/torrent_info.cpp

```diff
@@ -80,10 +80,10 @@ namespace libtorrent {
 	// this is an arbitrary limit to avoid malicious torrents causing
 	// unreasaonably large allocations for the merkle hash tree
 	// the size of the tree would be max_pieces * sizeof(int) * 2
-	// which is about 8 MB with this limit
+	// which is about 256 MB on a 64-bit system with this limit
```

Comment:
With`constexpr int default_piece_limit = 0x1000000;`, the max would be 128 MiB. `sizeof(int)` is still `4` on a 64-bit system (in practice, because in theory it can be anything).

Reply:
Thanks so much.  I fixed this.